### ISSUE TITLE
Design and implement revocation certificates

### ISSUE THREAD

**Issue by mikehelmick** [google/exposure-notifications-server#530]
### TL;DR

After receiving a publish request, respond with a _revocation certificate_

The revocation certificate can be used to later delete a set of published TEKs from the server's database. This does not remove the TEKs from already generated exports.

### Design

**Proposal**

On receiving a batch of TEKs - return an encrypted token that contains the those TEKs.

If that token is returned to the server, delete those TEKs from primary storage.

The token must be encrypted because anyone intercepting it should not be able to determine what the TEKs covered by it are.

/kind enhancement
/assign

**Comment by mikehelmick** [comment]
This is currently on hold. Will revisit if this becomes necessary.

/close
